_component_design/tables-basic.md

```diff
@@ -153,3 +155,71 @@ When using the check box icon within a table, each row supports the following st
 {% endcolumn %}
 
 </div>
+
+## Variations
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Dual line text table
+
+This variation of the basic table pattern is designed to allow for specialized content where cells need to display two lines of content.
+
+#### When to use
+
+This variation is used in the following situations: 
+
+- When a key has two values that are meaningfully related to each other and require display within the same cell. For example, last updated cell in the image to the left.
+- When two independent key:values are more meaningful displayed together.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Dual line text table example. **note to scale**"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/basic-tables/tables-dualline-overview.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+#### Best practices
+
+- **Rule**: Each cell with more than one line of text should have a designated Primary and Secondary value, this is important.  
```

Comment:
**Rule**: (**Important**) Each cell with more than one line of text must have a designated primary and secondary value.

Reply:
updated.